cristin-commons/src/main/java/no/unit/nva/cristin/common/handler/CristinQueryHandler.java

```diff
@@ -39,6 +39,7 @@ protected void validateQueryParameterKeys(RequestInfo requestInfo) throws BadReq
 
     protected String getValidPage(RequestInfo requestInfo) throws BadRequestException {
         var page = requestInfo.getQueryParameterOpt(PAGE).orElse(FIRST_PAGE);
+        requestInfo.getMultiValueQueryParameter()
```

Comment:
what does this do?

Reply:
Oops. Looked up some code and committed by mistake. Removed